pkg/live/inventoryrg.go

```diff
@@ -193,20 +199,52 @@ func (icm *InventoryResourceGroup) GetObject() (*unstructured.Unstructured, erro
 			"name":      objMeta.Name,
 		})
 	}
+	klog.V(4).Infof("Creating list of %d resources status", len(icm.objMetas))
+	var objStatus []interface{}
+	for _, objMeta := range icm.objMetas {
+		status, found := objStatusMap[objMeta]
+		if found {
+			klog.V(4).Infof("storing inventory obj refercence and its status: %s/%s", objMeta.Namespace, objMeta.Name)
+			objStatus = append(objStatus, map[string]interface{}{
+				"group":     objMeta.GroupKind.Group,
+				"kind":      objMeta.GroupKind.Kind,
+				"namespace": objMeta.Namespace,
+				"name":      objMeta.Name,
+				"status":    "UnKnown",
+				"strategy":  status.Strategy.String(),
+				"actuation": status.Actuation.String(),
+				"reconcile": status.Reconcile.String(),
+			})
+		}
+	}
+
 	// Create the inventory object by copying the template.
 	invCopy := icm.inv.DeepCopy()
 	// Adds or clears the inventory ObjMetadata to the ResourceGroup "spec.resources" section
 	if len(objs) == 0 {
 		klog.V(4).Infoln("clearing inventory resources")
 		unstructured.RemoveNestedField(invCopy.UnstructuredContent(),
 			"spec", "resources")
+		unstructured.RemoveNestedField(invCopy.UnstructuredContent(),
+			"status", "resourceStatuses")
 	} else {
 		klog.V(4).Infof("storing inventory (%d) resources", len(objs))
 		err := unstructured.SetNestedSlice(invCopy.UnstructuredContent(),
 			objs, "spec", "resources")
 		if err != nil {
 			return nil, err
 		}
+		err = unstructured.SetNestedSlice(invCopy.UnstructuredContent(),
+			objStatus, "status", "resourceStatuses")
```

Comment:
why did we call this `resourceStatuses` and not just `resources`? it's already under the `status` parent map.

Reply:
To make it clear that this field contains the status information for the resources.

Reply:
just seems redundant... but it's not new to this PR, so whatever.